packages/rolldown/build.ts

```diff
@@ -87,6 +87,7 @@ function withShared(
   return {
     input: {
       index: './src/index',
+      ...isBrowserBuild ? { config: './src/config' } : {},
       ...!isBrowserBuild
```

Comment:
Why only add the input to browser build?

Reply:
I think I assumed the browser build would be UMD and wouldn't support subpath imports, and had a brainfart and added the opposite...

Reply:
Browser build is ESM for bundlers like Vite, and without node built-in modules.